README.md

```diff
@@ -210,7 +210,6 @@ compress: {
   }
 }
 ```
-Note that `warnings` will be always overridden depending on the Grunt ["verbose" option](https://gruntjs.com/using-the-cli#verbose-v).
```

Comment:
This is not the case?
Commit message does not explain removal of this line.

Reply:
Yeah, missed to do that. From the Changelog v5.0.0-beta.0: `BREAKING: There are no more warnings being emitted`.